typescript/cli/ci-test.sh

```diff
@@ -150,6 +154,32 @@ run_hyperlane_deploy_core_dry_run() {
     AGENT_CONFIG_FILENAME=`ls -t1 /tmp | grep agent-config | head -1`
 }
 
+run_hyperlane_deploy_warp_dry_run() {
+    BEFORE_CORE_DRY_RUN=$(cast balance $DEPLOYER --rpc-url http://127.0.0.1:${CHAIN1_PORT});
```

Comment:
`BEFORE_CORE_DRY_RUN` -> `BEFORE_DRY_RUN` or `BEFORE_WARP_DRY_RUN`

Reply:
Ah yes, updated this since first commits. Was just a copy oversight from my other CI test– also ended up change artifact directories for both